core/templates/components/common-layout-directives/common-elements/confirm-or-cancel-modal.component.ts

```diff
@@ -22,7 +22,7 @@ import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
 export class ConfirmOrCancelModal {
   constructor(protected modalInstance: NgbActiveModal) {}
 
-  confirm<T>(value: T): void {
+  confirm<T>(value?: T): void {
```

Comment:
We were calling this function from some html templates without any params passed.

Reply:
done!

Reply:
Not done?

Reply:
Discussed with Sean and Vojta and this is now resolved.